web/packages/teleterm/src/ui/services/statePersistence/statePersistenceService.ts

```diff
@@ -31,14 +32,20 @@ interface UsageReportingState {
   askedForUserJobRole: boolean;
 }
 
+/**
+ * Expected shape of the persisted workspaces.
+ * In the future, it should come from zod.
+ */
+export type PersistedWorkspace = Omit<
+  Workspace,
+  'accessRequests' | 'documentsRestoredOrDiscarded' | 'color'
+> & { color?: ProfileColor };
```

Comment:
`DELETE IN 19.0.0`? As far as I understand this is just to express that existing persisted state from older versions might not have color defined?